app-misc/qlcplus/qlcplus-4.12.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils udev vcs-snapshot virtualx xdg
```

Comment:
This has a proper tarball, so the use of `vcs-snapshot` is dubious. If need be, set `S` manually.

Reply:
done